updater/templates/base/CHANGELOG.md

```diff
@@ -12,4 +12,4 @@ Bugfixes:
 
 Other improvements:
 
-## [0.0.0] - 2020-01-01
+
```

Comment:
Should we delete this line? Otherwise, will there be two lines after "Other improvements:"?

Reply:
Right, because the first release section will start on that second line.

In other words, current behavior will produce this:
```
Other improvements:

v4.0.2 - date

```
and removing the second line would produce this:
```
Other improvements:
v4.0.2 - date

```

Reply:
I see. I worry a little that formatters might inadvertently strip that line (it's common for tooling to trim all but one trailing newline in a file), and it might be more robust to insert that extra line in the `append` function. Otherwise someone might inadvertently (via a formatter) or intentionally delete the extra line without us all remembering why there are two newlines in the file. What do you think?

Reply:
In that case, let's just move the changelog template content into this function call entirely. AFAICT, there's no difference between `base` and `uses-js`, so that should resolve the issue.

Reply:
Latest commit implements this change. However, it will overwrite any previous CHANGELOG.md file without moving the previous file (if there is one) to the backups folder. I assume that isn't an issue for our purposes, but not sure whether we want to fix that now or later.

Reply:
Hmm. I think that _is_ an issue. What if we revert that change and just add the extra newline in the `append` function? That way the CHANGELOG is still backed up and it's clear from looking at the templates directory that this is a file that's going to be added.

Reply:
Ok, I've made the changes you suggested.